lib/searchDB.js

```diff
@@ -1,7 +1,14 @@
 const fs = require("fs-extra");
-var inputsFile = JSON.parse(fs.readFileSync('./json/matchForm.json', 'utf8'));
-var scoutIDs = JSON.parse(fs.readFileSync('./json/scouts.json', 'utf8'));
```

Comment:
Even though checks existed in `main.js` for `scouts.json`, `searchDB` was loaded at startup and thus attempted to load `scouts.json` before those tests and thus threw errors if the file didn't exist.